src/components/ViewWrapper.vue

```diff
@@ -1,15 +1,18 @@
 <template>
   <div>
+    <ConfirmDialogShared ref="confirm" attach="#app" />
```

Comment:
Sadly, App.vue also has this dialog, but fixing that will require some thought -- out of scope for now.

Reply:
(Or rather, we will look into this anyway when we try to simply App.vue to just a shell.)